tls/_constructs.py

```diff
@@ -222,7 +249,7 @@
                min_size=1, max_size=2 ** 16 - 1),
     Bytes("url", lambda ctx: ctx.length),
     TLSOneOf(UBInt8('padding'), [1]),
-    Bytes("sha1_hash", 20),
+    SHA1Hash,
```

Comment:
Since [TrustedAuthority's](https://tools.ietf.org/html/rfc6066#section-6) definition seems to borrow from [Client Certificate URLs'](https://tools.ietf.org/html/rfc6066#section-5) I went ahead and made them use the same `SHA1Hash` construct.